libs/vm/include/vm/variant.hpp

```diff
@@ -511,44 +511,61 @@ inline void Serialize(ByteArrayBuffer &buffer, Variant const &variant)
 
   switch (variant.type_id)
   {
+  case TypeIds::Unknown:
+  case TypeIds::Null:
+  case TypeIds::Void:
+    // No need to serialise the value (conceptually there is no value).
```

Comment:
Wouldn't the correct behaviour be to do `RuntimeError("Cannot serialize void/null/unknown")`?

Reply:
Left it in for the scenarios when someone instantiates `Variant` instance without specifying type and value, which is harmless - we can serialize & deserialise it with no effort or impact on functionality.

I can take it out.